src/components/layouts/DashBoardLayout/index.tsx

```diff
@@ -21,12 +22,17 @@ const imgUrl =
 
 export default function DashBoardLayout(props: DashBoardLayoutProps): React.ReactElement<DashBoardLayoutProps> {
   const { children, email, firstName, lastName, imageURL, pageTitle } = props;
-
+  const imgOptions = ['q_40'];
```

Comment:
What available image options are available on Cloudinary? I'm guessing a lot if so, let's set up agreed values for thumbnail sized avatar, full screen, medium-sized e.t.c
Once we have them we can put them in the `utils/constants` and everyone can use them easily.